src/tests/automerge_downtime_ok.py

```diff
@@ -116,8 +116,10 @@ def get_downtime_dict_at_version(sha, fname):
 
 def get_rg_resources_at_version(sha, fname):
     txt = get_file_at_version(sha, fname)
+    if not txt:
+        return {}
     rg = yaml.safe_load(txt)
-    return rg["Resources"]
+    return rg.get("Resources") or {}
```

Comment:
'get' takes a second parameter for what to use as the default.
```suggestion
     return rg.get("Resources", {})
```
Also, what if yaml.safe_load fails?

Reply:
I can add a non-exception-throwing wrapper around this which (more gently) writes some error message to stderr